test/test_integration.py

```diff
@@ -629,6 +638,11 @@ def render_content(tab):
         self.snapshot("Tabs with Graph - initial (graph should not resize)")
         tab_two.click()
 
+        # wait for content to be loaded after clicking
+        WebDriverWait(self.driver, 10).until(
+            EC.visibility_of_element_located((By.ID, "tabs-content-example"))
```

Comment:
should you be looking for data that's actually inside the tab rather than the tab container itself? We might even want to wait for the graph to be rendered. i.e. something like `#graph-`-tabs .svg-container`. Similarly below when you switch tabs (`#graph-2-tabs .svg-container`)